mmedit/models/backbones/sr_backbones/rdn.py

```diff
@@ -0,0 +1,187 @@
+import torch
+from mmcv.runner import load_checkpoint
+from torch import nn
+
+from mmedit.models.registry import BACKBONES
+from mmedit.utils import get_root_logger
+
+
+class DenseLayer(nn.Module):
+    """Dense layer
+
+    Args:
+        in_channels (int): Channel number of inputs.
+        out_channels (int): Channel number of outputs.
+
+    """
+
+    def __init__(self, in_channels, out_channels):
+        super(DenseLayer, self).__init__()
+        self.conv = nn.Conv2d(
+            in_channels, out_channels, kernel_size=3, padding=3 // 2)
+        self.relu = nn.ReLU(inplace=True)
+
+    def forward(self, x):
+        """Forward function.
+
+        Args:
+            x (Tensor): Input tensor with shape (n, c, h, w).
+
+        Returns:
+            Tensor: Forward results, tensor with shape (n, c, h, w).
+        """
+        return torch.cat([x, self.relu(self.conv(x))], 1)
+
+
+class RDB(nn.Module):
+    """Residual Dense Block of Residual Dense Network
+
+    Args:
+        in_channels (int): Channel number of inputs.
+        out_channels (int): Channel number of outputs.
+    """
+
+    def __init__(self, in_channels, growth_rate, num_layers):
+        super(RDB, self).__init__()
+        self.layers = nn.Sequential(*[
+            DenseLayer(in_channels + growth_rate * i, growth_rate)
```

Comment:
is growth_rate an int or a float ratio?

Reply:
It is `int`, which is the channel number of dense layer output.